### DISCUSSION TITLE
Trying to clean room

### DISCUSSION THREAD

**Discussion by sagidam** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#430] [General]
I have lydsto r1 
Cleaning all house work perfect
I get error service not found, only when i am trying to use room cleaning, or spot cleaning
Cant find the right solution?

**Comment by PiotrMachowski** [comment]
What integration do you use?

**Comment by sagidam** [comment]
Xiaomi Miot Auto

**Reply by PiotrMachowski** [reply]
Please do not respond via email, it doesn't work with threads.

You have to figure out config for your vacuum by yourself, here are example configs for different models:
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/317

**Reply by sagidam** [reply]
Evrery thing work great except the service call, I see all the rooms on map and the area is corect, why i get failed to call service, service not found, what i need to change, the config for my vacum is here 
https://home.miot-spec.com/spec/zhimi.vacuum.xa1
What do you think?

**Reply by PiotrMachowski** [reply]
Oh, I have pasted a wrong link. Here's the correct one:
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/406

**Reply by sagidam** [reply]
Can you add the configuration to lydsto R1 
https://home.miot-spec.com/spec/zhimi.vacuum.xa1
I cant understand the configuration

**Reply by PiotrMachowski** [reply]
I can't add it, because I don't know it. You can try to figure it by comparing it to already existing configs
